spec/src/main/asciidoc/configexamples.asciidoc

```diff
@@ -121,6 +121,11 @@ public class InjectedConfigUsageSample {
     @ConfigProperty(name="myprj.some.supplier.timeout", defaultValue="100")
     private java.util.function.Supplier<Long> timeout;
 
+    //Injects a Map to resolve multiple configuration parameters under the same configuration key.
```

Comment:
For the Map injection, what types for Key, Value do you propose to support? We need to clarify that.